packages/recommend/src/types/TrendingFacetsQuery.ts

```diff
@@ -1,3 +1,6 @@
 import { TrendingQuery } from './TrendingQuery';
 
-export type TrendingFacetsQuery = Omit<TrendingQuery, 'model' | 'facetValue'>;
+export type TrendingFacetsQuery = Omit<
+  TrendingQuery,
+  'model' | 'facetValue' | 'fallbackParameters' | 'queryParameters'
```

Comment:
At this point why have a shared `TrendingQuery` type if they differ so much?

According to the API endpoints:

- Trending facets
  - `facetName`: required
- Trending items
  - `facetName`: optional
  - `facetValue`: optional

Reply:
Yeah that's a good point

Reply:
yeah, i wanted to do that in a separate PR, both for client and for the libs